### PR TITLE
Docsp 8639 migrate mongodb tools to new commmandline repo

### PR DESCRIPTION
**Pull request by andf-mongodb**

### CODE REVIEW

File: source/includes/extracts-4.2-changes.yaml [hunk 1]
```diff
@@ -0,0 +1,842 @@
+ref: 4.2-changes-index-names
```
Comment (rkumar-mongo) [inline source/includes/extracts-4.2-changes.yaml]: Hm - given that the new tools are 4.4 specific, how often are we referencing this content? I don't see a lot of refs linking here.

File: source/includes/extracts-download-from-tools.yaml [hunk 2]
```diff
@@ -0,0 +1,101 @@
+ref: _download-from-tools
+content: |
+  The {{command}} tool is part of the MongoDB tools package. Consult the
```
Comment (rkumar-mongo) [inline source/includes/extracts-download-from-tools.yaml]: Small warning - it appears that for the latest 4.3 dev series there's just no package for the tools. Or at least not one I Can find. hopefully they resolve that soon!

File: source/index.txt [hunk 3]
```diff
@@ -1,13 +1,169 @@
...
+:binary:`~bin.mongostat` and :binary:`~bin.mongotop`
+provide diagnostic information related to the current operation of a
+:binary:`~bin.mongod` instance.
+
+:binary:`~bin.mongoreplay` is a traffic capture and replay tool for MongoDB
+that you can use to inspect and record commands sent to a MongoDB
+instance, and then replay those commands back onto another host at a
+later time.  :binary:`~bin.mongoreplay` is available for Linux and macOS.
+
+MongoDB 3.4 provides :binary:`~bin.mongoldap` for testing native operating system
```
Comment (rkumar-mongo) [inline source/index.txt]: mongoldap isn't included in the commandline tools, to my knowledge. See https://github.com/mongodb/mongo-tools

File: source/index.txt [hunk 4]
```diff
@@ -1,13 +1,169 @@
...
+
+- :doc:`/mongofiles`
+
+.. toctree::
+   :maxdepth: 1
+   :hidden:
+
+   /mongofiles
+
+MongoDB Compass
```
Comment (rkumar-mongo) [inline source/index.txt]: I don't think we need Compass here - its not a commandline tool.

File: source/install_compass.txt [hunk 5]
```diff
@@ -0,0 +1,100 @@
+.. _install-compass:
```
Comment (rkumar-mongo) [inline source/install_compass.txt]: Same thought w.r.t. Compass - its not a commandline tool. If anything, after we finish migration we would leave this as part of the MongoDB server installation side of things.

File: source/mongo.txt [hunk 6]
```diff
@@ -0,0 +1,1249 @@
+.. _mongo:
```
Comment (rkumar-mongo) [inline source/mongo.txt]: I'm pretty sure the mongo shell is its own package, and is not in the tools package.

File: source/mongos.exe.txt [hunk 7]
```diff
@@ -0,0 +1,115 @@
+==============
+``mongos.exe``
```
Comment (rkumar-mongo) [inline source/mongos.exe.txt]: Same as w/ mongo shell - mongos is its on package, and not in tools.

File: source/mongos.txt [hunk 8]
```diff
@@ -0,0 +1,1990 @@
+.. _mongos:
```
Comment (rkumar-mongo) [inline source/mongos.txt]: See comment on mongos.exe

File: source/includes/extracts-4.2-changes.yaml (hunk 1 above)
Comment (rkumar-mongo) [reply source/includes/extracts-4.2-changes.yaml]: ping on this - what happens if we remove this file?

File: source/index.txt [hunk 9]
```diff
@@ -1,13 +1,104 @@
 :orphan:
 
-=======================================
-The MongoDB |version| Dev Series Manual
-=======================================
+================================================================
+The MongoDB |version| Dev Series Commandline Tools Documentation
```
Comment (rkumar-mongo) [inline source/index.txt]: Note to change this to a source constant:

`{+server-utils+}`

And add a new field to the `source_constants` object in `conf.py` ->

`'server-utils' : 'Server Utilities'`

**Comment by rkumar-mongo** [comment]
Two remaining fix ups, then lets merge this so we can start cleaning up a bit.

File: source/includes/extracts-4.2-changes.yaml (hunk 1 above)
Comment (rkumar-mongo) [reply source/includes/extracts-4.2-changes.yaml]: cc @andf-mongodb

Comment (andf-mongodb) [reply source/includes/extracts-4.2-changes.yaml]: It's referenced 9 times. 1:1 transfer for now imo.

File: source/index.txt (hunk 9 above)
Comment (andf-mongodb) [reply source/index.txt]: Great minds. But Snooty doesn't appear to permit source constants in titles. I've done this everywhere except for the title, which I left hard-coded.

File: source/includes/extracts-4.2-changes.yaml (hunk 1 above)
Comment (rkumar-mongo) [reply source/includes/extracts-4.2-changes.yaml]: Ah damn. We'll tidy up later then.

**Comment by andf-mongodb** [comment]
Done.

**Pull request closed without merging**